projects/packages/videopress/src/client/admin/components/admin-page/index.tsx

```diff
@@ -66,22 +69,28 @@ const useDashboardVideos = () => {
 		videos,
 		localVideos,
 		uploadedVideoCount,
-		localTotalVideoCount,
+		uploadedLocalVideoCount,
 		hasVideos,
 		hasLocalVideos,
 		handleFilesUpload,
+		handleLocalVideoUpload,
 		loading: isFetching,
+		uploading: uploading?.length > 0,
```

Comment:
`uploading` should be a boolean, as it is defined here. The original `uploading` from `useVideo` should be something more descriptive like `videosBeingUploaded`.
Not requesting change, just a random comment.

Reply:
it should be a selector directly from state, this way we could access it around the places